specs/parser/lexical/lists_spec.js

```diff
@@ -1,36 +1,32 @@
-import Immutable from 'immutable';
 import { parserFor } from '../../../app/oz/parser';
 import lexicalGrammar from '../../../app/oz/grammar/lexical.nearley';
+import lexical from '../../samples/lexical';
 
 const parse = parserFor(lexicalGrammar);
 
 describe('Parsing lists', () => {
-  beforeEach(() => {
-    jasmine.addCustomEqualityTester(Immutable.is);
+  it('handles empty lists correctly', () => {
+    expect(parse('nil')).toEqual(lexical.nil());
```

Comment:
This is picked up by the atom parser, right?. If so, we don't really need to check that parsing `nil` is handled properly because `nil` is an atom.

Reply:
ok, this will be removed.